static/sass/_pattern_navigation.scss

```diff
@@ -200,6 +200,10 @@ $meganav-height: 3rem;
     }
 
     .p-navigation__row--25-75 {
+      .p-navigation__items:first-child {
```

Comment:
Why is this needed? `p-navigation__row--25-75` was added specifically to avoid need for custom styling to match the grid.

Reply:
Ah, ok, it's because you override the left padding on the nav items, so you need to compensate.